server/README.md

```diff
@@ -22,23 +22,23 @@ To be able to open Keycloak on localhost map port 8080 locally
 
 ## Creating admin account
 
-To be able to login to the admin console you need to have admin user.  
+To be able to login to the admin console you need to have an admin user.  
 To create an admin account with your login and password you need to use environment variables to pass in an 
 initial username and password. This is done by running:
 
     docker run -e KEYCLOAK_USER=<USERNAME> -e KEYCLOAK_PASSWORD=<PASSWORD> jboss/keycloak
 
-When you run the container without `KEYCLOAK_USER` and `KEYCLOAK_PASSWORD` user and password are generated 
-automatically, you can find it with:
+When you run the container without the `KEYCLOAK_USER` and `KEYCLOAK_PASSWORD` variables, the username and password 
+of the admin user are generated automatically. You can find the actual username / password of the admin account with:
     
     docker logs <CONTAINER> | grep 'Your first login and password' -A3
 
-You will find next lines:
+You will see output similar to the following one:
     
     docker logs  <CONTAINER> | grep 'Your first login and password' -A3
         Your first login and password:
```

Comment:
@14DF397FCA Thank you for your changes! 

Please replace ```Your first login and password:``` above in the docs yet with ```Username and password of the Keycloak admin account:``` so the docs is consistent with the code.

And replace ```KEYCLOAK_USER``` with ```Login:```, and ```KEYCLOAK_PASSWORD``` with ```Password:``` strings below, so it's again consistent with the code.

Sorry, if being a way too much pedantic :(

Reply:
@iankko , I updated the request.

Don't worry)